ibis/backends/impala/__init__.py

```diff
@@ -266,7 +266,7 @@ def raw_sql(self, query: str):
     def _fetch_from_cursor(self, cursor, schema):
         from ibis.formats.pandas import PandasData
 
-        results = fetchall(cursor)
+        results = fetchall(cursor, schema.names)
```

Comment:
The cursor names aren't case sensitive, which can lead to a name mismatch. Just use the schema names when we have a schema.